### ISSUE TITLE
Screen Reader not announcing title of Explore Data pages

### ISSUE THREAD

**Issue by benhammondmusic** [SatcherInstitute/health-equity-tracker#1055]
- [x] Explore the data	SR	[A11Y - Health Equity Tracker]: Improper announcement when user navigates between subpages under Explore the Data page_Web_1.16(c)	"ENVIRONMENT:
macOS 11.3 | Chrome Version 90.0.4430.212 (Official Build) (x86_64) | Voiceover
Chrome OS Version 91.0.4472.60 (Official Build) beta (64-bit) | Chromevox

PRE REQUISITES:
Turn on the respective screen reader

STEPS TO REPRODUCE:
1). Open https://healthequitytracker.org/exploredata
2). Navigate to the all the three subpages under Explore the Data
3). Observe the announcement

EXPECTED RESULT:
Page should have a unique title and screen reader should announce it properly. 
 
OBSERVED RESULT:
Screen reader doesn't announce a title when user navigates among the subpages of Explore the data page. When user navigates among the subpages, screen reader announces 'Compare across race and ethnicity group'.

Replication video attached"	https://drive.google.com/file/d/1qSPAWSAkULc-9mmVywldnOvzz0Sul8dl/view?usp=sharing	"Issue is replicable in the following Android environment:

Device: Pixel 4
Android Version: 11
Chrome Browser Version: 91.0.4472.101"

- [x] Explore the data	SR	[A11Y - Health Equity Tracker]: Page title not announced when user navigates to Explore Data page_Web_2.6	"ENVIRONMENT:
macOS 11.3 | Chrome Version 90.0.4430.212 (Official Build) (x86_64) | Voiceover
Chrome OS Version 91.0.4472.60 (Official Build) beta (64-bit) | Chromevox

PRE REQUISITES:
Turn on the respective screen reader

STEPS TO REPRODUCE:
1). Open https://healthequitytracker.org/
2). Navigate to Explore the Data link and activate the link
3). Observe the announcement

EXPECTED RESULT:
Page should have a unique title and screen reader should announce it properly. 
 
OBSERVED RESULT:
Screen reader doesn't announce the page title when user navigates to Explore the data page. It announces some content from the page when user navigates to Explore the Data page.

Replication video attached"	https://drive.google.com/file/d/19Y88uqQImtlmgWVXizKQsJpNtMYqQKeN/view?usp=sharing

**Comment by benhammondmusic** [comment]
Ideally now that we have helmet we can render specific titles for the 3 different carousel settings; helpful for screen reader users

**Comment by benhammondmusic** [comment]
Initial load working like other pages; changes to madlib / carousel result in an alert to SR user #1389
